ckan/cli/jobs.py

```diff
@@ -15,8 +15,11 @@ def jobs():
 
 @jobs.command(short_help=u"Start a worker.",)
 @click.option(u"--burst", is_flag=True, help=u"Start worker in burst mode.")
+@click.option(u"--max-idle-time", default=None, type=click.INT,
```

Comment:
can't set it to half a second?

Reply:
Nope, cannot do doubles or floats. So no fraction of seconds, but I do not think that is really needed